### ISSUE TITLE
Importing ABCs directly from collections deprecated in Python 3.7, stopped working in 3.8

### ISSUE THREAD

**Issue by danielschenk** [Xion/callee#12]
When I run some tests with pytest and callee 0.3, this is printed at the end:
```
============================== warnings summary ===============================
c:\users\rnd10\appdata\local\programs\python\python37-32\lib\site-packages\callee\collections.py:74
  c:\users\rnd10\appdata\local\programs\python\python37-32\lib\site-packages\callee\collections.py:74: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    CLASS = collections.Iterable

c:\users\rnd10\appdata\local\programs\python\python37-32\lib\site-packages\callee\collections.py:116
  c:\users\rnd10\appdata\local\programs\python\python37-32\lib\site-packages\callee\collections.py:116: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    CLASS = collections.Sequence

c:\users\rnd10\appdata\local\programs\python\python37-32\lib\site-packages\callee\collections.py:128
  c:\users\rnd10\appdata\local\programs\python\python37-32\lib\site-packages\callee\collections.py:128: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    CLASS = collections.Set

c:\users\rnd10\appdata\local\programs\python\python37-32\lib\site-packages\callee\collections.py:246
  c:\users\rnd10\appdata\local\programs\python\python37-32\lib\site-packages\callee\collections.py:246: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    CLASS = collections.Mapping

-- Docs: https://docs.pytest.org/en/latest/warnings.html
======================= 78 passed, 4 warnings in 1.38s ========================
```
Pytest version info:
```
platform win32 -- Python 3.7.3, pytest-5.1.0, py-1.8.0, pluggy-0.12.0
plugins: cov-2.7.1, ordering-0.6, pycharm-0.6.0
```
Solution suggestions: https://stackoverflow.com/questions/53978542/how-to-use-collections-abc-from-both-python-3-8-and-python-2-7

**Comment by danielschenk** [comment]
Just noticed there is already a fix and pull request for this: https://github.com/Xion/callee/pull/10

**Comment by Xion** [comment]
As mentioned above, this is fixed with PR #10.
